localstack/constants.py

```diff
@@ -26,6 +26,7 @@
 DEFAULT_PORT_EVENTS_BACKEND = 4554
 DEFAULT_PORT_LOGS_BACKEND = 4553
 DEFAULT_PORT_SECRETSMANAGER = 4552
+DEFAULT_PORT_SES = 4551
```

Comment:
Can we please change lines 28-29 to:
```
DEFAULT_PORT_SECRETSMANAGER_BACKEND = 4552
DEFAULT_PORT_SES_BACKEND = 4551
```
The reason is that these constants define the port numbers of the backend services, whereas the default ports of the frontend API endpoints (proxy endpoints) are defined here: https://github.com/localstack/localstack-python-client/blob/master/localstack_client/config.py#L5-L64